### DISCUSSION TITLE
2024/12/20/giscus%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F-%E6%97%A0%E6%B3%95%E6%AD%A3%E5%B8%B8%E6%98%BE%E7%A4%BA-%E8%AF%84%E8%AE%BA%E5%8F%91%E4%B8%8D%E5%87%BA-%E8%AF%84%E8%AE%BA%E5%8F%91%E5%87%BA%E4%B8%8D%E6%98%BE%E7%A4%BA%E9%97%AE%E9%A2%98/

### DISCUSSION THREAD

**Discussion by giscus** [Torae233/Torae233.github.io#24] [Announcements]
# 2024/12/20/giscus%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F-%E6%97%A0%E6%B3%95%E6%AD%A3%E5%B8%B8%E6%98%BE%E7%A4%BA-%E8%AF%84%E8%AE%BA%E5%8F%91%E4%B8%8D%E5%87%BA-%E8%AF%84%E8%AE%BA%E5%8F%91%E5%87%BA%E4%B8%8D%E6%98%BE%E7%A4%BA%E9%97%AE%E9%A2%98/

一个热爱音乐，爱玩FPS和Galgame的无产阶级革命者

http://localhost:4000/2024/12/20/giscus%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F-%E6%97%A0%E6%B3%95%E6%AD%A3%E5%B8%B8%E6%98%BE%E7%A4%BA-%E8%AF%84%E8%AE%BA%E5%8F%91%E4%B8%8D%E5%87%BA-%E8%AF%84%E8%AE%BA%E5%8F%91%E5%87%BA%E4%B8%8D%E6%98%BE%E7%A4%BA%E9%97%AE%E9%A2%98/

**Comment by Teio96** [comment]
我也是...问问解决了嘛

**Reply by Teio96** [reply]
解决了，发现是缩进问题！

**Reply by Torae233** [reply]
哥们能展开细讲一下吗？拜托了

**Reply by Teio96** [reply]
我是后来发现在主题的配置文件里评论区部分的配置代码缩进有问题🤣lz也可以看看

**Comment by hahahotsoup** [comment]
我的解决方式有点怪...trae自动把categories改了，并且后续我自己改成announcement没加复数...我自己和我自己左右脑互搏了属于是
